shaptools/shapcli.py

```diff
@@ -326,18 +338,23 @@ def run():
     logger = setup_logger(args.verbosity or logging.DEBUG)
 
     if args.config:
```

Comment:
we should perhaps comment what are we checking with this if for reducing the cognitive load